R/http.R

```diff
@@ -130,8 +134,9 @@ TWIT_paginate_max_id <- function(token, api, params,
     ) 
     withr::defer(pb$terminate())
   }
-
-  for (i in seq_len(pages)) {
+  
+  i<-0
+  while ((i<-i+1) <= pages) {
```

Comment:
I think it's easier to understand if you do the check and the increment separately.

Reply:
yes, probably easier. changed this.